recipes/drogon/all/conanfile.py

```diff
@@ -154,6 +161,7 @@ def generate(self):
         tc.variables["BUILD_MYSQL"] = self.options.get_safe("with_mysql", False)
         tc.variables["BUILD_SQLITE"] = self.options.get_safe("with_sqlite", False)
         tc.variables["BUILD_REDIS"] = self.options.get_safe("with_redis", False)
+        tc.variables["USE_SPDLOG"] = self.options.get_safe("with_spdlog", False)
```

Comment:
```suggestion
        if Version(self.version) > "1.9.1":
            tc.variables["USE_SPDLOG"] = self.options.get_safe("with_spdlog", False)
```
So we avoid cmake warnings to older versions.